modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/CreateAPIApplication.testcase

```diff
@@ -85,6 +85,11 @@ definition {
 		task ("Then the test API is created successfully") {
 			Alert.viewSuccessMessageText(successMessage = "New API Application was created.");
 
+			ApplicationsMenu.gotoPortlet(
```

Comment:
@magjed4289 I don't think so, there is a new action, when you create an API Application, it will go to application's edit page
[Screencast from 2023年08月25日 15时17分31秒.webm](https://github.com/liferay-headless/liferay-portal/assets/89900202/f0a6697b-8c47-4553-97fd-7c614937ed3f)

Reply:
the behaviour has changed then, cause in my bundle it staus on the list:

https://github.com/liferay-headless/liferay-portal/assets/35199951/bb6320fa-0df7-4852-ba56-a6dbd394fba5

Maybe it would be good to ask whether the change is intended.

Reply:
> then our test should be modified to see the Edit Screen 
:+1: